docs/reference/ingest/apis/enrich/put-enrich-policy.asciidoc

```diff
@@ -65,26 +65,10 @@ If you use {es} {security-features}, you must have:
 ==== {api-description-title}
 
 Use the put enrich policy API
-to create a new enrich policy.
-
-// tag::enrich-policy-def[]
-An *enrich policy* is a set of rules the enrich processor uses
-to append the appropriate data to incoming documents.
-An enrich policy contains:
-
-* The *policy type*,
-  which determines how the processor enriches incoming documents
-* A list of source indices
-* The *match field* used to match incoming documents
-* *Enrich fields* appended to incoming documents
-  from matching documents
-// end::enrich-policy-def[]
-
-
-===== Update an enrich policy
+to create a new <<enrich-policy,enrich policy>>.
 
 // tag::update-enrich-policy[]
-You cannot update an existing enrich policy.
+Once created, you can't update or change an enrich policy.
 Instead, you can:
 
 .   Create and execute a new enrich policy.
```

Comment:
here and in the general "enriching your data" docs, there is a lot of discussion about executing policies, but we never link to the Execute Policy API. I know it is already in the list of actions, but I think it would be nice to hyperlink to the rest API action. what do you think?

Reply:
I added some additional cross-links with 65716c6.